### PR TITLE
Fix formatting and replace tabs with spaces

### PR DESCRIPTION
**Pull request by telotortium**

### CODE REVIEW

File: org-gcal.el [hunk 1]
```diff
@@ -391,8 +391,8 @@ needed."
                ("grant_type" . "refresh_token"))
        :parser 'org-gcal--json-read
        :error
-       (cl-function (lambda (&key error-thrown &allow-other-keys)
-                    (message "Got error: %S" error-thrown))))
+       (cl-function (lambda (&key error-thrown &allow-other-keys))
```
Comment (kidd) [inline org-gcal.el]: I think there are some moved parenthesis like here.

**Comment by kidd** [comment]
Thanks for that! 
I think that there are some moved paretheses. Did you really mean to move them?

**Comment by telotortium** [comment]
I think I've fixed all the moved parens now - please take a look.

**Comment by kidd** [comment]
yes! thanks!

**Pull request merged**